Resources/doc/reference/action_list.rst

```diff
@@ -52,23 +52,25 @@ Here is an example from Sonata E-Commerce Product Admin:
     public function configureListFields(ListMapper $listMapper)
     {
         $listMapper
-            // addIdentifier allows to specify that this column will provide a link to the entity's edition
+            // addIdentifier allows to specify that this column
+            // will provide a link to the entity's edition
```

Comment:
IMO edit or show is not correct, because it depends on your rights, if you have edit rights, you will be redirected to the editAction, otherwise to the showAction...

but ```will provide a link to the entity```